commands/create-account.js

```diff
@@ -117,6 +117,19 @@ async function createAccount(options) {
     } else {
         await eventtracking.track(eventtracking.EVENT_ID_CREATE_ACCOUNT_END, { success: true, new_keypair: false }, options);
     }
+    // Check to see if account already exists
+    try {
+        // This is expected to error because the account shouldn't exist
+        await near.account(options.accountId);
+        console.error(`Sorry, account '${options.accountId}' already exists.`);
+        return;
```

Comment:
should return failure exit code, this returns success

Reply:
pushed, this was an oversight on my part, whoops. thanks for catching

Reply:
nit: probably makes sense to throw vs call `process.exit` directly (to be consistent with other errors)

Reply:
there isn't an actual error here, though, right? This is in the `try` section.

<img width="732" alt="Screenshot 2020-07-10 15 11 54" src="https://user-images.githubusercontent.com/1042667/87207006-b9cd4100-c2bf-11ea-98bc-26f3b40e50c5.png">

Reply:
what I mean is that we can do:

```
throw new Error(`Sorry, account '${options.accountId}' already exists.`)
```

so that we hit normal error handling path. It's not like super important by itself but makes stuff easier if we want to do some common stuff on error (like report it to analytics).